### PR TITLE
[zh] - docs/reference/command-line-tools-reference/feature-gates.md

### PR DESCRIPTION
**Pull request by zhangguanzhang**
Signed-off-by: zhangguanzhang <zhangguanzhang@qq.com>
Ref: https://github.com/kubernetes/website/issues/29327#issuecomment-898237476

### CODE REVIEW

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -1034,7 +1199,8 @@ Each feature gate is designed for enabling/disabling a specific feature:
 -->
 - `KubeletPodResources`：启用 kubelet 的 Pod 资源 GRPC 端点。更多详细信息，请参见
   [支持设备监控](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/compute-device-assignment.md)。
-- `KubeletPodResourcesGetAllocatable`：启用 kubelet 的 pod 资源 `GetAllocatableResources` 功能。
+- `KubeletPodResourcesGetAllocatable`：启用 kubelet 的 pod 资源
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:1202]:
```suggestion
- `KubeletPodResourcesGetAllocatable`：启用 kubelet 上 pod 资源
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 2]
```diff
@@ -1224,6 +1419,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
 -->
 - `SCTPSupport`：在 Pod、Service、Endpoints、NetworkPolicy 定义中
   允许将 _SCTP_ 用作 `protocol` 值。
+- `SeccompDefault`: 允许使用 `RuntimeDefault` 来为所有工作附着在设置默认的 seccomp 配置文件。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:1422]:
```suggestion
- `SeccompDefault`: 允许将所有工作负载的默认  seccomp 配置文件为 `RuntimeDefault`。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 3]
```diff
@@ -1224,6 +1419,9 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
+   seccomp 配置在 Pod 或者容器的 `securityContext` 字段中指定。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:1423]:
```suggestion
  seccomp 配置在 Pod 或者容器的 `securityContext` 字段中指定。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 4]
```diff
@@ -1275,6 +1475,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
 -->
 - `StartupProbe`：在 kubelet 中启用
   [启动探针](/zh/docs/concepts/workloads/pods/pod-lifecycle/#when-should-you-use-a-startup-probe)。
+- `StatefulSetMinReadySeconds`: 允许 StatefulSet 控制器遵守 `minReadySeconds`。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:1478]:
```suggestion
- `StatefulSetMinReadySeconds`: 允许 StatefulSet 控制器采纳 `minReadySeconds` 设置。
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 5]
```diff
@@ -1375,29 +1576,31 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
-  [Enabling Endpoint Slices](/docs/tasks/administer-cluster/enabling-endpointslices/).
 -->
 - `VolumeSubpathEnvExpansion`：启用 `subPathExpr` 字段用于将环境变量在 `subPath`
   中展开。
 - `WarningHeaders`：允许在 API 响应中发送警告头部。
 - `WatchBookmark`：启用对 watch 操作中 bookmark 事件的支持。
 - `WinDSR`：允许 kube-proxy 为 Windows 创建 DSR 负载均衡。
 - `WinOverlay`：允许 kube-proxy 在 Windows 的覆盖网络模式下运行。
+- `WindowsEndpointSliceProxying`: 当启用时，运行在 Windows 上的 kube-proxy
+  将使用 EndpointSlices 而不是 endpoint 作为主要数据源，从而实现可伸缩性和性能改进。
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:1597]:
```suggestion
  将使用 EndpointSlices 而不是 Endpoints 作为主要数据源，从而实现可伸缩性和并改进性能。 
```

File: content/zh/docs/reference/command-line-tools-reference/feature-gates.md [hunk 6]
```diff
@@ -1375,29 +1576,31 @@ Each feature gate is designed for enabling/disabling a specific feature:
...
 -->
 - `VolumeSubpathEnvExpansion`：启用 `subPathExpr` 字段用于将环境变量在 `subPath`
   中展开。
 - `WarningHeaders`：允许在 API 响应中发送警告头部。
 - `WatchBookmark`：启用对 watch 操作中 bookmark 事件的支持。
 - `WinDSR`：允许 kube-proxy 为 Windows 创建 DSR 负载均衡。
 - `WinOverlay`：允许 kube-proxy 在 Windows 的覆盖网络模式下运行。
+- `WindowsEndpointSliceProxying`: 当启用时，运行在 Windows 上的 kube-proxy
+  将使用 EndpointSlices 而不是 endpoint 作为主要数据源，从而实现可伸缩性和性能改进。 
+  详情请参见[Enabling Endpoint Slices](/zh/docs/concepts/administer-cluster/enabling-endpointslices/).
```
Comment (tengqm) [inline content/zh/docs/reference/command-line-tools-reference/feature-gates.md:1598]:
```suggestion
  详情请参见[启用短点切片](/zh/docs/concepts/administer-cluster/enabling-endpointslices/).
```

**Comment by zhangguanzhang** [comment]
all had done

**Comment by zhangguanzhang** [comment]
@chenrui333 PTAL

**Pull request merged**